CoreScriptsRoot/ServerCoreScripts/ServerSocialScript.lua

```diff
@@ -38,7 +38,6 @@ local function decodeJSON(json)
 		return HttpService:JSONDecode(json)
 	end)
 	if not success then
-		print("decodeJSON() failed because", result, "Input:", json)
```

Comment:
I'd like to keep this because it doesn't happen often and when we get errors it's good to log them